src/player.py

```diff
@@ -27,17 +27,17 @@ def add_to_trait(self, name, value):
         """Add 'value' to 'trait'"""
         pass
 
-    def add_items(self, items):
+    def change_items(self, items):
         """Add all items in 'items' (dictionary name to value) in the player's inventory"""
-        pass
+        for item, count in items.items():
+            self.items[item] = max(self.items.get(item, 0) + count, 0)
```

Comment:
just make a quick comment to say "this will clamp the value to 0 if .get returns a negative value."